cmd/juno/juno.go

```diff
@@ -25,21 +26,23 @@ const greeting = `
 Juno is a Go implementation of a Starknet full node client created by Nethermind.`
 
 const (
-	configF   = "config"
-	logLevelF = "log-level"
-	rpcPortF  = "rpc-port"
-	dbPathF   = "db-path"
-	networkF  = "network"
-	ethNodeF  = "eth-node"
-	pprofF    = "pprof"
-	colourF   = "colour"
-
-	defaultConfig  = ""
-	defaultRPCPort = uint16(6060)
-	defaultDBPath  = ""
-	defaultEthNode = ""
-	defaultPprof   = false
-	defaultColour  = true
+	configF              = "config"
+	logLevelF            = "log-level"
+	rpcPortF             = "rpc-port"
+	dbPathF              = "db-path"
+	networkF             = "network"
+	ethNodeF             = "eth-node"
+	pprofF               = "pprof"
+	colourF              = "colour"
+	pendingPollIntervalF = "pending-poll-interval"
+
+	defaultConfig              = ""
+	defaultRPCPort             = uint16(6060)
+	defaultDBPath              = ""
+	defaultEthNode             = ""
+	defaultPprof               = false
+	defaultColour              = true
+	defaultPendingPollInterval = time.Duration(0)
```

Comment:
I think the default should be more than 0. Maybe 1s? Otherwise, the ticker will go crazy, if the user forgets to assign the poll interval.

Reply:
Nevermind. Just saw that it is disabled by default.